include/stdio.h

```diff
@@ -46,6 +46,11 @@
 #define SEEK_END 2
 #endif
 
+#define FOPEN_MAX 16
+#define L_ctermid 9
```

Comment:
strange value, please explain in the comment why it's valid

Reply:
FOPEN_MAX is described on opengroup page as follows - Number of streams which the implementation guarantees can be open simultaneously. The value is at least eight.

Open group says about L_ctermid - The symbolic constant L_ctermid is defined in [<stdio.h>], and shall have a value greater than 0.
 
On my ubuntu is defined as 9.

Reply:
I meant - add comment in code :).

This function defines the maximum output len for `ctermid()` function (without `\0`) - I know it's not implemented but it's directly related to device naming convention in our OS. It needs to be able to hold UART device paths  (so `/dev/uartXX` assuming no more than 99 uarts on single HW) and pseudoterminal paths (`/dev/pts/XX...`) defined in `posixsrv` (see https://github.com/phoenix-rtos/phoenix-rtos-posixsrv/blob/master/pty.c#L613). Right now even `/dev/pts/1` would exceed Your limit.

Please make sure it fits at least `/dev/pts/XX` and write so in comment in the code.